cmd/tsconnect/common.go

```diff
@@ -111,3 +118,30 @@ type EsbuildMetadata struct {
 		EntryPoint string `json:"entryPoint,omitempty"`
 	} `json:"outputs,omitempty"`
 }
+
+func setupEsbuildTailwind(build esbuild.PluginBuild, dev bool) {
+	build.OnLoad(esbuild.OnLoadOptions{
+		Filter: "./src/index.css$",
+	}, func(args esbuild.OnLoadArgs) (esbuild.OnLoadResult, error) {
+		start := time.Now()
+		yarnArgs := []string{"--silent", "tailwind", "-i", args.Path}
```

Comment:
I assume esbuild handles all the file watching for us here?

It may later end up being faster to rely on tailwind's own `--watch` flag, but this seems like a fine place to start.

Reply:
esbuild invokes this every time the CSS is requested, so it adds a bit of latency. Managing a persistent watch process will add some complexity, we look into it if the slowdown becomes noticeable during development.